client/src/components/Common/BookingForm/BookingForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import Calender from '../../../Images/calendar_icon.png'
+import Clock from '../../../Images/clock_icon.png'
+import RoomIcon from '../../../Images/room_icon.png'
+import './bookingForm.css'
+import axios from 'axios'
+import { Link } from 'react-router-dom'
+import Popup from './Popup'
+class BookingForm extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      userId : 3,
+      date: '2020-05-23T11:00:00.000Z',
+      roomId:  5,
+      room_name : 'Moscow' ,
+      toggelReminder: false ,
+      showPopup : false
+    }
+  }
+
+  handelChange = (e) => {
+    const name = e.target.name
+    const value =e.target.value
+    this.setState({[name]: value})
+  }
+  handelChecked = ()=> {
+      this.setState({toggelReminder : !this.state.toggelReminder })
+  }
+
+   handelSubmit =(e) => {
+      e.preventDefault();
+      const formData = this.state
+      axios.post( '/api/booking', formData)
+      .then(response => {if(response.status===200){
+        console.log( 'res :',response)
+       return this.setState({showPopup : !this.state.showPopup})}
+      // else  alert('A small error happend in the server in this booking operation!! try again !!')
+      })
+      .catch(err => console.log(err))
+   }
+
+  render () {
+    return (
+      <div>
+        <div className="date_info_continer__div">
+          <div className="date_info__div">
+            <img src={Calender} alt='calender' />
+            <h4>{this.state.date.slice(0,10)}</h4>
```

Comment:
Just FYI, parsing and slicing the datetime like this is a bit dangerous as it is relying on a very specific format of the input, which might not always hold true, or might change in the future. A better way would be to parse it through `Date.parse()` or some 3rd party lib and then use it's API to display the data you want. But don't waste time on it now, it's not the most important thing..